cmd/bbolt/main.go

```diff
@@ -456,7 +456,7 @@ func (cmd *pageItemCommand) Run(args ...string) error {
 	fs := flag.NewFlagSet("", flag.ContinueOnError)
 	fs.BoolVar(&options.keyOnly, "key-only", false, "Print only the key")
 	fs.BoolVar(&options.valueOnly, "value-only", false, "Print only the value")
-	fs.StringVar(&options.format, "format", "ascii-encoded", "Output format. One of: "+FORMAT_MODES)
+	fs.StringVar(&options.format, "format", "auto", "Output format. One of: "+FORMAT_MODES)
```

Comment:
@ahrtr Hello ✋🏽 

So to test this change, I ran the `TestPageItemCommand_Run` and it passes. 

currently the test case assert the keys and values are being inserted in https://github.com/etcd-io/bbolt/blob/4154f9c642333237ac659a560a30bfb37802e4c2/cmd/bbolt/main_test.go#L143-L146 exists within the page, it capture the `Main.Stdout` as a string here  https://github.com/etcd-io/bbolt/blob/4154f9c642333237ac659a560a30bfb37802e4c2/cmd/bbolt/main_test.go#L165 

Although the key and value are being inserted are `int` typed, they are asserted as a string prefix 

I am stuck in what should I assert to test the changes ? 

I can insert `hex` data for instance, but still I will capture the output as a `string`, please correct me if I am wrong ?